src/server/api/controllers/gameInstances.controller.js

```diff
@@ -0,0 +1,16 @@
+const knex = require('../../config/db');
+
+const getGameInstances = async () => {
+  try {
+    return await knex('game_instances')
+      .join('users', 'users.id', 'game_instances.fk_user_id')
+      .select('game_instances.id', 'users.name', 'game_instances.score')
```

Comment:
Hey lucha according to the endpoint you have to create you had to select all the game_instances records of ongoing games. So the validation is correct but i would like to see the fields of the game_instances here. Thanks 🙂

Reply:
Hi @priyankapandey13 I considered some information to be irrelevant for the users, like the FK. 
But it´s true that the response could be more complete, so I added the times and the new field **code** created in the commit 6aaff5a30f0609bde8454cee5795e6f3795d5a08
 Thanks!

Feedback implemented in commit 38e01fad4476fb5039ea778aeda7e323b97e64c4 :)

Reply:
I agree with you, code looks good but what i didn't get is "game_instances.code". I ran the code and matched the DB diagram. There is a mistake in the tables we have. In db diagram code is a part of "game_instance" table and  score is a part of game_scores. I pulled the code and setup a new db with your help yesterday so it must be the updated one. but the tables i have score is the part of "game_instances" table. So i think it needs fixing....